app/decorators/physical_chassis_decorator.rb

```diff
@@ -0,0 +1,39 @@
+class PhysicalChassisDecorator < MiqDecorator
+  def self.fonticon
+    'pficon pficon-cluster'
+  end
+
+  def fileicon
+    "svg/vendor-#{vendor.downcase}.svg"
+  end
+
+  def quadicon
+    {
+      :top_left     => {
+        :text    => physical_servers.count,
+        :tooltip => _('Servers')
```

Comment:
`_('Physical Servers')˙

Reply:
Or even better, I'm revisiting all the tooltips in the quads and you can be faster by using them right now :wink: 
Something like this:
https://github.com/skateman/manageiq-ui-classic/blob/quadicon-tooltips/app/decorators/manageiq/providers/infra_manager_decorator.rb#L17-L18